.github/workflows/jepsen-cluster-txm.yml

```diff
@@ -0,0 +1,65 @@
+name: jepsen-cluster-txm
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I suggest to change strategy here, not run long jepsen etst om each push (which is way to troubles) but rather accumulate patches and run jepsen via cron. 

i.e. for fast jepsen test to run jepsen once 30 minutes, and for longer multi-node tests - run it once an hour or 2;

(Certainly checking that test is already invoked for particular hash, to avoid unnecessary runs if there is no activity that period)

Reply:
Author of changes is responsible for passing tests when we are running tests triggered by push. 
When we trigger tests by cron someone from a test team should process fails, create bug report, find a guilty and so on. 
So I'm against approach with cron.

Reply:
Unfortunately there is no way to make pipeline of workflows. Right, there is the '[workflows_runs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run)' trigger exists, but it can't check the status of all previous workflows finished - it will receive the same event of success completion from each of the workflow it waits, [check](https://github.community/t/workflow-run-completed-event-triggered-by-failed-workflow/128001/7?u=avtikhon):
```
The workflow_run event occurs when a workflow run is requested or completed, and allows you to execute a workflow based on the finished result of another workflow.

example
on:
workflow_run:
  workflows: [ test ]
  types: 
    - completed
However by itself, this doesn’t quite work as expected.

The completed type, does not indicate success, for that you’d have to include the following in each job of your workflow:

if: ${{ github.event.workflow_run.conclusion == 'success' }}
If you’re depending on more than one workflow, then ANY of them completing, will trigger the event

example
name: deploy

on:
workflow_run:
  workflows: [ test, lint, compile ]
  types: 
    - completed
if your test workflow fails, but lint completed successfully, github.event.workflow_run.conclusion == 'success' will still be true

Your workflow will trigger as many times as you have workflow dependencies

in the previous example, our deploy workflow, will run 3 times!

All this makes the workflow_run event fundamentally broken for any advanced usage
```

Reply:
So, I'm thinking about cron way as the only best solution in our situation.